examples/MultipleRedeemers.purs

```diff
@@ -0,0 +1,254 @@
+        alwaysMintsPolicy
+      cs :: CurrencySymbol <-
+        liftContractM "could not get curencysymbol from mintingpolicy" $
+          mkCurrencySymbol <<< getCurrencySymbol =<< scriptCurrencySymbol mp
+      -- TODO: It would probably make sense to incorporate an actual Script instead of the alwayssucceeds 
+      --       one as someone could always just spend our tokens while we wait between the transactions
+      tok :: Array (Tuple TokenName Int) <-
+        liftContractM "could not make tokennames with values" $ traverse
+          (bitraverse (mkTokenName <=< byteArrayFromAscii) pure)
+          tokens
+
+      logInfo' $ "trying to create" <> show tok
+
+      let
+        toCsValue :: Array (Tuple TokenName Int) -> Value.Value
+        toCsValue t =
+          unwrap <<< toPlutusType <<< Value mempty <<< NonAdaAsset
+            <<< Map.singleton cs $ Map.fromFoldable $ map fromInt <$> t
+
+        tokenCount :: Int
+        tokenCount = sum $ snd <$> tokens
+
+        values :: Value.Value
+        values = toCsValue tok
+
+        lookups :: Lookups.ScriptLookups PlutusData
+        lookups = mconcat
+          [ Lookups.validator validator
+          , Lookups.mintingPolicy mp
+          ]
+
+        constraints :: Constraints.TxConstraints Unit Unit
+        constraints = mconcat
+          [ Constraints.mustMintValue values
+          -- create all the tokens in one utxo each
+          , mconcat $
+              ( \x -> replicate tokenCount
+                  $ Constraints.mustPayToScript vhash unitDatum
+                  $ toCsValue
+                  $ pure
+                  $ Tuple (fst x) 1
+              ) =<< tok
+          ]
+
+      ubTx <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
+      BalancedSignedTransaction bsTx <-
+        liftedM "Failed to balance/sign tx" $ balanceAndSignTx ubTx
+
+      logInfo' $ "Balanced and signed tx is " <> show bsTx
+      logInfo' $ "Created utxos " <> show values
+
+      Tuple <$> submit bsTx.signedTxCbor <*> pure tokenCount
+
+  log $ "Created utxos with transactionhash " <> show hash
+  log "Going on with spending scriptoutputs from previous transaction"
+
+  let
+    getAlwaysSucceedsUtxos
+      :: forall (r :: Row Type)
+       . Contract r (Tuple Utxo (Array TransactionInput))
+    getAlwaysSucceedsUtxos = do
+      utxos :: UtxoM <-
+        liftContractM "could not get utxos at alwaysSucceedsScript" =<< utxosAt
+          (scriptHashAddress vhash)
+      liftAff $ delay $ Milliseconds $ toNumber 3_000
+      let
+        orefs :: Array TransactionInput
+        orefs = Set.toUnfoldable
+          $ Set.filter ((_ == hash) <<< _.transactionId <<< unwrap)
+          $ Map.keys
+          $ unwrap utxos
+
+      if (length orefs == numTokens) then
+        pure $ Tuple (unwrap utxos) orefs
+      else do
+        logInfo' "Could not find utxos, trying again"
+        getAlwaysSucceedsUtxos
+
+  runContract cfg $ do
```

Comment:
```suggestion
  runContract_ cfg $ do
```

Reply:
But actually this should be moved anyway (see my comment about a single `runContract_` call above)